hazelcast/src/main/java/com/hazelcast/query/impl/Index.java

```diff
@@ -23,47 +23,106 @@
 import java.util.Set;
 
 /**
- * This interface contains the methods related to index of Query.
+ * Represents an index built on top of the attribute of the map entries.
```

Comment:
maybe "on top of an attribute of the map values"?

Reply:
Too late, already merged, will create another PR today. "Entries" are legitimate, AFAIU we may create indexes on key attributes also. The article should be fixed.

Reply:
Right, key attributes are also a valid target for indexing.